src/firmware/boards/robot_stm32h7/io/allegro_a3931_motor_driver.c

```diff
@@ -1,41 +1,34 @@
 #include "firmware/boards/robot_stm32h7/io/allegro_a3931_motor_driver.h"
 
+#include <stdbool.h>
 #include <stdlib.h>
 
 typedef struct AllegroA3931MotorDriver
 {
     PwmPin_t* pwm_pin;
     GpioPin_t* reset_pin;
-    GpioPin_t* coast_pin;
     GpioPin_t* mode_pin;
     GpioPin_t* direction_pin;
-    GpioPin_t* brake_pin;
-    GpioPin_t* esf_pin;
+    bool enabled;
 } AllegroA3931MotorDriver_t;
 
-AllegroA3931MotorDriver_t* io_allegro_a3931_motor_driver_create(
-    PwmPin_t* pwm_pin, GpioPin_t* reset_pin, GpioPin_t* coast_pin, GpioPin_t* mode_pin,
-    GpioPin_t* direction_pin, GpioPin_t* brake_pin, GpioPin_t* esf_pin)
+AllegroA3931MotorDriver_t* io_allegro_a3931_motor_driver_create(PwmPin_t* pwm_pin,
+                                                                GpioPin_t* reset_pin,
+                                                                GpioPin_t* mode_pin,
+                                                                GpioPin_t* direction_pin)
```

Comment:
_interesting_, why'd we remove the pins? just don't need them?

Reply:
yup, coast and brake are now just handled by pulling reset high and setting pwm to 0 respectively

the new motor driver rev doesn't expose those pins anymore